Ensure that the exitCode is != 0 in case of Errors. (#120)
Draegerwerk/SDCcc: README.md

| R0056           | TriggerDescriptorUpdate |
| R0078_0         | TriggerReport           |

## Exit Codes

SDCcc's exitCode should be interpreted as follows:

| **ExitCode** | **Semantics**                                                                                                        |
|--------------|----------------------------------------------------------------------------------------------------------------------|

> | 0            | Success - SDC-Compliance could be successfully measured and the Device Under Test satisfies all tested Requirements  |
> | 1            | Failure - SDC-Compliance could be successfully measured, but the Device Under Test violated Requirements             |
> | 2            | Error - SDC-Compliance could not be measured                                                                         |

Reviewer: The phrasing is odd, especially measurement. Shouldn't we go for something like
```suggestion
| 0            | Success - Test run execution was successful and the device under test satisfies all tested requirements  |
| 1            | Failure - Test run execution was successful, but the device under test violated requirements             |
| 2            | Error - Test run execution was not successful                                                                         |
```
